app/src/androidTest/java/edu/uc/group/rankine/ExampleInstrumentedTest.kt

```diff
@@ -1,13 +1,11 @@
 package edu.uc.group.rankine
 
-import androidx.test.platform.app.InstrumentationRegistry
 import androidx.test.ext.junit.runners.AndroidJUnit4
-
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.Assert.*
 import org.junit.Test
 import org.junit.runner.RunWith
 
-import org.junit.Assert.*
-
```

Comment:
Deleted unnecessary imports
Source: https://rules.sonarsource.com/kotlin/RSPEC-125

Reply:
It looks like they are all still here... just re-ordered.